op-supervisor/supervisor/backend/syncnode/node.go

```diff
@@ -407,6 +411,10 @@ func (m *ManagedNode) onReplaceBlock(replacement types.BlockReplacement) {
 		ChainID:     m.chainID,
 		Replacement: replacement,
 	})
+	// if the node replaced a block, both the unsafe and safe are reset to this point
+	m.lastNodeLocalSafe = replacement.Replacement.ID()
```

Comment:
Was this causing an observable issue or just an oversight that was noticed?

Reply:
This was causing the logs we saw in those E2E tests you were doing, where after a replacement it claims to need a reset. It isn't clear to me if a reset would have *actually* been done, but it definitely doesn't need to.